include/openssl/httperr.h

```diff
@@ -17,39 +17,43 @@
 # include <openssl/cryptoerr_legacy.h>
 
 
+# ifndef OPENSSL_NO_HTTP
+
```

Comment:
This is technically an API break. Please revert this. The reason code defines should not be removed by disabling the respective feature.